.gitignore

```diff
@@ -20,13 +20,7 @@
 /tmp
 
 # Ignore public precompiled assets
-/public/assets
-
-# Ignore precompiled static error pages
-/public/404.html
-/public/422.html
-/public/500.html
-/public/503.html
+/public
```

Comment:
I think you wanna revert this one because there are things in /public that we do want in the repo like the favicon.ico, robots.txt and the fonts

Reply:
these were on my local filesystem 🤔 
I can remove this change and not check them in.. 

OH - I think it's because I was locally trying to debug the compile_asset command and built them with the production environment 🤦🏼 - I'll remove.